README.md

```diff
@@ -36,11 +36,52 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
 
 To contribute new features and changes to the website, you would want to run the app locally. Please follow these steps:
 
+
+## How to setup the project in their local machine
+
 1. Clone the project locally. Make sure you have a recent version of Node.js installed on your computer.
 
-2. You can use the included [generated.json](generated.json) as dummy data or you can run `npm run prebuild` to fetch the latest data from GitHub yourself: for this, you will need to set the `GH_PERSONAL_ACCESS_TOKEN` environment variable to a valid GitHub Personal Access Token (PAT). Notice: repositories not maching the criteria listed above (see rules in [generated.json](generated.json)) are automatically removed from [firstissue.json](firstissue.json) when the [generated.json](generated.json) script runs.
+2. Fork the repository
+Forking the repository means creating a copy of the original repository in your Github account. To do this, navigate to the repository you want to contribute to and click on the “Fork” button in the top-right corner.
```

Comment:
This is not necessary for just running the project locally. Also, forking should be done before step 1, therefore first fork, then clone.